packages/web/src/components/DevFatalErrorPage.tsx

```diff
@@ -241,11 +243,7 @@ function ResponseRequest(props: { error: ErrorWithRequestMeta }) {
                 onClick={() => setOpenQuery(!openQuery)}
                 className={openQuery ? 'open' : 'preview'}
               >
-                {JSON.stringify(
-                  props.error.mostRecentRequest.query,
-                  null,
-                  '  '
-                )}
+                {print(props.error.mostRecentRequest.query)}
```

Comment:
IMO, this should probably happen inside the apollo provider - my Relay provider doesn't require this for example

Reply:
Aha. I see. Well maybe that will solve some of the bundle size issue too.

Reply:
I'm marking that back to Draft as we revisit the bundle and where the printing is needed.

Reply:
Have moved this into Apollo